regression/cbmc/SIMD1/test.desc

```diff
@@ -0,0 +1,8 @@
+CORE gcc-only broken-smt-backend
```

Comment:
Still broken after #5974? Also, there is an `#ifdef _MSC_VER` in the test, which suggests it might work on Visual Studio?

Reply:
The SMT conversion still fails (at `src/solvers/smt2/smt2_conv.cpp:2531 function: convert_typecast`), but Visual Studio should indeed be ok. Fixed.